doc/source/scalability/innodb_io.rst

```diff
@@ -54,17 +54,16 @@ The following values are allowed:
     use ``fsync()`` to flush both the data and log files.
 
   * ``O_SYNC``: 
-    use O_SYNC to open and flush the log files; use ``fsync()`` to flush the data files.
+    use O_SYNC to open and flush the log files; use ``fsync()`` to flush the data files. When this value is set :ref:`parallel_doublewrite_buffer` file is opened with ``O_SYNC`` flag. 
```

Comment:
use O_SYNC to open and flush the log and parallel doublewrite files; use fsync() to flush the data files. Do not use fsync() to flush the parallel doublewrite file.